### ISSUE TITLE
A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_ScreenReader:The name of the dialog is not announced as visually seen.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9177]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#NVDA;#JAWS;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: NVDA,JAWS
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Launch screen reader as NVDA .
4.  Hover over the 'Servers' header and select the 'New Server Group' icon.

**Actual:**
When the dialog opens screen reader announces it as "Server Group" dialog

**Same issue with JAWS as well.**

**Expected:**
Screen reader should read the name as "Add Server group" dialog as seen on screen.

**User Impact:**
Screen reader users will not receive the right name of the dialog, they will not know that the dialog is for adding server groups.

**Attachment link for Reference** :
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EbvhPuL-AaJNipzHIGuCcUEBSL_cNnuONBeSpjcJ1upm1Q?e=mmN8Eb

**Comment by alanrenmsft** [comment]
dupe of #9228

**Comment by Roopesh-T** [comment]
![image](https://user-images.githubusercontent.com/86601127/217234779-5cfc0cb7-ba63-4a0e-81a8-ff31d3596606.png)
The above issue is not repro in the env
![image](https://user-images.githubusercontent.com/86601127/217234978-2b8906af-1b03-4f1f-83c5-3004fe9ebbda.png)
so closing the bug.

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
